kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/ListAllReassignmentsResponse.java

```diff
@@ -19,10 +19,6 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 import com.google.auto.value.AutoValue;
 
-/**
- * Response body for {@code GET /v3/clusters/<clusterId>/topics/-/partitions/-/reassignments}
- * requests.
- */
 @AutoValue
 public abstract class ListAllReassignmentsResponse {
```

Comment:
This class can actually be renamed to just ListReassignmentsResponse. The "all" is valid for request  - `GET /v3/clusters/<clusterId>/topics/-/partitions/-/reassignments`

Reply:
I think you should have three classes:

* `ListAllReassignmentsResponse` for `/v3/clusters/<clusterId>/topics/-/partitions/-/reassignments` (#682)
* `SearchReassignmentsByTopicResponse` for `/v3/clusters/<clusterId>/topics/<topicName>/partitions/-/reassignments` (#687)
* `GetReassignmentsResponse` for `/v3/clusters/<clusterId>/topics/<topicName>/partitions/<partitionId>/reassignments` (this PR)

Reply:
That sounds good, updated #687 with separate response class as `SearchReassignmentsByTopicResponse`.